runtime/test/demo-flow-test.js

```diff
@@ -36,6 +36,7 @@ describe('demo flow', function() {
     planner.init(arc);
 
     let plans = await planner.suggest();
+    debugger;
```

Comment:
Did you want to remove that before submit?

Reply:
ups, fixed.